src/containers/pages/FocusInvestigation/map/active/utils.ts

```diff
@@ -0,0 +1,126 @@
+import superset from '@onaio/superset-connector';
+import {
+  SUPERSET_GOALS_SLICE,
+  SUPERSET_JURISDICTIONS_SLICE,
+  SUPERSET_MAX_RECORDS,
+  SUPERSET_PLANS_SLICE,
+  SUPERSET_STRUCTURES_SLICE,
+  SUPERSET_TASKS_SLICE,
+} from '../../../../../configs/env';
+import { AN_ERROR_OCCURED } from '../../../../../configs/lang';
+import { JURISDICTION_ID, PLAN_ID } from '../../../../../constants';
+import { displayError } from '../../../../../helpers/errors';
+import { popupHandler } from '../../../../../helpers/handlers';
+import supersetFetch from '../../../../../services/superset';
+import { fetchGoals, Goal } from '../../../../../store/ducks/goals';
+import { fetchJurisdictions, Jurisdiction } from '../../../../../store/ducks/jurisdictions';
+import { fetchPlans, Plan } from '../../../../../store/ducks/plans';
+import { setStructures, Structure } from '../../../../../store/ducks/structures';
+import { fetchTasks, Task } from '../../../../../store/ducks/tasks';
+
+/**
+ * Fetch data for the plan
+ * @param fetchGoalsActionCreator // fetch goals action creator
+ * @param fetchJurisdictionsActionCreator // fetch jurisdication action creator
+ * @param fetchPlansActionCreator // fetch plans action creator
+ * @param fetchStructuresActionCreator // fetch structures action creator
+ * @param fetchTasksActionCreator // fetch tasks action creator
+ * @param plan // plan object
+ * @param supersetService // fetch superset promise
+ */
+export const fetchData = async (
```

Comment:
What are your thoughts about using displayError here as opposed to throwing the errors and letting them be handled by whoever calls this function?